File: engine/model/src/main/scala/tools/packet/foreigneffect/ShowChair.scala
package tools.packet.foreigneffect

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class ShowChair(private var _characterId: Int, private var _itemId: Int) extends PacketInput {
  def characterId: Int = _characterId

  def itemId: Int = _itemId

  override def opcode(): SendOpcode = SendOpcode.SHOW_CHAIR
}

File: engine/engine-model/src/main/java/tools/packet/foreigneffect/ShowChair.java
package tools.packet.foreigneffect;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record ShowChair(Integer characterId, Integer itemId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.SHOW_CHAIR;
   }
}